play/src/front/WebRtc/CoWebsite/SimpleCoWebsite.ts

```diff
@@ -60,7 +54,7 @@ export class SimpleCoWebsite implements CoWebsite {
 
     load(): CancelablePromise<HTMLIFrameElement> {
         this.loadIframe = new CancelablePromise((resolve, reject, cancel) => {
```

Comment:
This function should be useless now.
Move the content to the svelte component instead.